packages/twenty-front/src/modules/workspace-member/utils/formatDateLabel.ts

```diff
@@ -0,0 +1,35 @@
+import { DateFormat } from '@/workspace-member/constants/DateFormat';
+import { detectDateFormat } from '@/workspace-member/utils/detectDateFormat';
+import { WorkspaceMemberDateFormatEnum } from '~/generated/graphql';
+
+export const getDateFormatFromWorkspaceEnum = (
+  dateLabel: WorkspaceMemberDateFormatEnum,
+) => {
+  switch (dateLabel) {
+    case WorkspaceMemberDateFormatEnum.System:
+      return detectDateFormat();
```

Comment:
Have you checked the performance / how often this was recomputed?

Reply:
(e.g. basic console.log into that function ; if it's called more than once upon loading the initial page then we're probably doing something wrong - I didn't test)

Reply:
I don't think there is any problem here, its called only once 👍